### PR TITLE
Remove path encoding for UI Browse tab

### LINKED ISSUE #9506: Remove URLEncoding in browse field
**Alluxio Version:**
2.1.0-SNAPSHOT

**Describe the bug**
In the browse GO field, paths will be URL encoded.

**To Reproduce**
Go to the browse tab and navigate to a directory, the new browse textbox will show `%2F` instead of `/`

**Expected behavior**
The path in the input box is not URL encoded.

### PR DESCRIPTION
**Pull request by tieujason330**
- Fixes #9506 
- make sure parsed path comes in decoded or else it'll get into an unreliable state

### CODE REVIEW

**Comment by gpang** [comment]
@tieujason330 can you be a bit more descriptive with the title (avoid using "fix")? The issue title is "Remove URLEncoding in browse field", so could this title mention removing the url encoding, if that is in fact what this is doing? Thanks!

File: webui/master/src/containers/pages/Browse/Browse.tsx [hunk 1]
```diff
@@ -73,6 +73,7 @@ export class Browse extends React.Component<AllProps, IBrowseState> {
     super(props);
 
     let {path, offset, limit, end} = parseQuerystring(this.props.location.search);
+    path = decodeURIComponent(path || '');
```
Comment (calvinjia) [inline webui/master/src/containers/pages/Browse/Browse.tsx]: Should the default value be `/`? 2 lines down we try to set path with `|| '/'`

File: webui/master/src/containers/pages/Browse/Browse.tsx [hunk 2]
```diff
@@ -215,7 +215,7 @@ export class Browse extends React.Component<AllProps, IBrowseState> {
                    `/browse?path=${path}${queryStringSuffix}`)}/>
           </FormGroup>
           <FormGroup className="mb-2 mr-sm-2">
-            <Button tag={Link} to={`/browse?path=${encodeURIComponent(path || "/")}${queryStringSuffix}`}
+            <Button tag={Link} to={`/browse?path=${encodeURIComponent(path || '/')}${queryStringSuffix}`}
```
Comment (calvinjia) [inline webui/master/src/containers/pages/Browse/Browse.tsx]: Unrelated to this PR, is it possible in typescript to infer that path cannot be undefined here?

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

File: webui/master/src/containers/pages/Browse/Browse.tsx (hunk 2 above)
Comment (tieujason330) [reply webui/master/src/containers/pages/Browse/Browse.tsx]: it depends how `path` is defined. In this case, it's defined as an optional string (`path?: string`) so typescript complains about that. I can look into simplifying this if possible

**Pull request merged**